auth-tokens/src/main/java/com/palantir/tokens/auth/BearerTokens.java

```diff
@@ -78,7 +78,7 @@ public static BearerToken fromPath(Path path) throws IOException {
         }
 
         LOGGER.warn("No token file found in any of the configured paths: {}.", paths);
-        return Optional.absent();
+        return Optional.empty();
```

Comment:
+1 to the proposal to move back to `tokens` package. I've spent a significant portion of my time on the conversion between v1 and v2 tokens while migrating a service to remoting3, and I'd imagine every team will have to do this until all service are migrated to remoting3.